src/containers/Layout/Sidebar.js

```diff
@@ -425,6 +426,14 @@ function Sidebar({ history, settings, setSetting, getGovernanceVenus }) {
             <Label primary>History</Label>
           </div>
         </NavLink>
+        <NavLink
+          className="flex flex-start align-center"
+          to="/career"
+          activeClassName="active"
+        >
+          <img className="career-icon" src={CareerIcon} alt="career" />
+          <Label primary>Career</Label>
```

Comment:
I believe plural is more common, see, e.g., binance.com
<img width="203" alt="Screenshot 2021-11-10 at 11 06 22" src="https://user-images.githubusercontent.com/10987782/141074308-86d36fb9-6513-4ba3-82b8-2aa7a1a4b398.png">

```suggestion
          <Label primary>Careers</Label>
```